src/libraries/System.Private.Xml/src/System/Xml/Schema/XmlSchemaValidator.cs

```diff
@@ -906,13 +934,12 @@ private void ValidateWhitespace(string elementStringValue, XmlValueGetter elemen
             }
         }
 
-
-        public object ValidateEndElement(XmlSchemaInfo schemaInfo)
+        public object? ValidateEndElement(XmlSchemaInfo schemaInfo)
```

Comment:
Seems `XmlSchemaInfo schemaInfo` is nullable for all public methods in this class